test/test_restful.py

```diff
@@ -699,6 +699,40 @@ def test_033_cv_agent_get(self):
         self.assertIn("port", json_response["results"], "Malformed response body!")
 
 
+    def test_034_cv_agent_post_invalid_exclude_list(self):
+        """Test CV's POST /v2/agents/{UUID} Interface"""
+        self.assertIsNotNone(self.V, "Required value not set.  Previous step may have failed?")
+
+        b64_v = base64.b64encode(self.V)
+        # Set unsupported regex in exclude list
+        ima_whitelist = { 'exclude': ['*'] }
```

Comment:
Is this right? You're loading the exclude list into the whitelist?

Reply:
This is how the CV REST API currently works. That is, the CV expects an `ima_whitelist` entry that consists of 2 list entries: a `whitelist` and `exclude` entry for each list. This code is just setting it up according to that. The tenant does something similar shown in the below snippets. We should probably consider separating both lists out into separate fields in the future.
- https://github.com/keylime/keylime/blob/4b7be7c56ab1b2e549e3d470710fb0f74780b197/keylime/ima.py#L273
- https://github.com/keylime/keylime/blob/4b7be7c56ab1b2e549e3d470710fb0f74780b197/keylime/tenant.py#L230-L231
- https://github.com/keylime/keylime/blob/4b7be7c56ab1b2e549e3d470710fb0f74780b197/keylime/tenant.py#L478